pandas/tests/indexes/test_numeric.py

```diff
@@ -105,6 +105,21 @@ def test_numeric_compat(self):
         for r, e in zip(result, expected):
             tm.assert_index_equal(r, e)
 
+        #test power calculations both ways
```

Comment:
model this after the existing code, IOW, use ``idx`` which is already define.

add a comment with this issue number.